core/editor/src/tool/tools/select.rs

```diff
@@ -186,6 +222,60 @@ impl Fsm for SelectToolFsmState {
 	}
 }
 
+enum AlignDimension {
+	X,
+	Y,
+}
+
+enum AlignAggregate {
+	Min,
+	Max,
+	Average,
+}
+
+fn align_selected(document: &Document, responses: &mut VecDeque<Message>, dimension: AlignDimension, aggregate: AlignAggregate) {
```

Comment:
It definitely seems like this should be a document function, because we should be able to call it from other parts of the codebase (from the file menu, from a future command search panel like in Blender, from a future console, etc.) without the Select Tool active.

Reply:
Agreed, done